hangups/ui/__main__.py

```diff
@@ -153,8 +152,11 @@ def _on_connect(self):
             yield from hangups.build_user_conversation_list(self._client)
         )
         self._conv_list.on_event.add_observer(self._on_event)
-        if not self._disable_notifier:
-            self._notifier = Notifier(self._conv_list)
+
+        # Connect the notifier
+        self._notifier._conv_list = self._conv_list
```

Comment:
Having to initialize `Notifier._conv_list` in a separate step isn't ideal. Another way we would work around this is have `ChatUI` manually call `self._notifier.on_event(conv, conv_event)` from its own event handler method. That way `Notifier` doesn't need `_conv_list` at all.